### ISSUE TITLE
BodyKinematics and PointKinematics acceleration calculation issues

### ISSUE THREAD

**Issue by carmichaelong** [opensim-org/opensim-core#2782]
See: https://simtk.org/plugins/phpBB/viewtopicPhpbb.php?f=91&t=11925&p=33869&start=0&view=

**Comment by carmichaelong** [comment]
With the pendulum model, also tried using `/|com_acceleration` with an `OutputReporter` which was in agreement with the erroneous `BodyKinematics` and `PointKinematics` acceleration calculations

**Comment by jenhicks** [comment]
There is not a technical issue, but confusing to users ... should clarify in documentation.

**Comment by carmichaelong** [comment]
> But from what I can tell the accelerations are not calculated as the second derivative of the input positions in the .mot file, but rather using the forces in the model (ie realizeAcceleration).

You're absolutely correct that this is how it's done. If you don't have the full set of constraints, external forces, etc. that drive a motion, you'll get unexpected answers. We've chatted about how to deal with this, and it might be that you could flip a flag for a separate mode to get accelerations from derivatives only. The other competing factor is that we were hoping to slowly move away from Analyses too... @clnsmith any suggestions from a user end is super helpful here.
